.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: go
 
 go:
-  - 1.4.3
-  - 1.5.2
+  - 1.10.3
 
+before_install:
+  - go get -t -v ./ably/...
 env:
   - GOMAXPROCS=4
-
 sudo: false
+script:
+  - make test
```

Comment:
Trailing newline needed here?

Reply:
nope, that is valid yaml and travis is happy with it. I can add it if you think it will serve any purpose.